### PR TITLE
Add missing required parameter in LifecycleNode launch action

### PR DESCRIPTION
**Pull request by ivanpauno**
See https://github.com/ros2/launch_ros/pull/157.

### CODE REVIEW

File: lifecycle/launch/lifecycle_demo.launch.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 def generate_launch_description():
     return LaunchDescription([
         LifecycleNode(package='lifecycle', executable='lifecycle_talker',
-                      name='lc_talker', output='screen'),
+                      name='lc_talker', namespace='', output='screen'),
```
Comment (sloretz) [inline lifecycle/launch/lifecycle_demo.launch.py:23]: Is `namespace=''` the same as `namespace='/'`? If so, the latter seems clearer to me

Comment (ivanpauno) [reply lifecycle/launch/lifecycle_demo.launch.py:23]: It's the same in this case.
The difference is that `''` can be modified when the launch file is being included in another launch file.

```python3
LaunchDescription([
  ...
  PushRosNamespace('my_ns'),
  IncludeLaunchDescription(path_to_lifecycle_demo_launch_py),
  ...
])
```

See examples of the difference here https://github.com/ros2/launch_ros/blob/master/test_launch_ros/test/test_launch_ros/actions/test_push_ros_namespace.py.

**Comment by ivanpauno** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/11261/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6521/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/9220/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/11169/)

**Comment by ivanpauno** [comment]
@ros-pull-request-builder retest this please

**Pull request merged**